app/src/main/res/values/strings.xml

```diff
@@ -128,7 +128,8 @@
     <string name="nc_settings_link_previews_desc">Allows previews of content from received links for supported services</string>
     <string name="nc_settings_link_previews_key" translatable="false">link_previews</string>
     <string name="nc_settings_read_privacy_key" translatable="false">read_privacy</string>
-    <string name="nc_locked">Tap to unlock</string>
+    <string name="nc_locked_tap_to_unlock">Tap to unlock</string>
+    <string name="nc_locked">locked</string>
```

Comment:
Should this be `Locked`? Or `translatable="false"`, unsure. :)

Reply:
It should be ```Locked``` 👍